modules/candidate_parameters/jsx/CandidateInfo.js

```diff
@@ -214,6 +242,15 @@ var CandidateInfo = React.createClass(
         }
       }
 
+      var snacks = {
+        1: 'Pretzel',
+        57: 'Goldfish',
+        15: 'Cheezball',
+        25: 'Gummi worm',
+        33: 'Bugle',
+        45: 'Fruit rollup',
+        102: 'Fruit by the foot'
```

Comment:
Question: can you add new snacks or they need to be already in the dropdown?

Reply:
you can add / define your own snacks IFF `strictSearch` is false, or you can make a pull request to this branch and add your own snacks to this object yourself :rice_ball: :pineapple: :fish_cake:

Reply:
makes sense